docs/flash_instructions.txt

```diff
@@ -0,0 +1,41 @@
+How to put devices into Bootsel mode
```

Comment:
Maybe a short description of Bootsel mode? I.e., the device / controller will show up as a drive.